pkg/authorization/authorizer/scope/converter.go

```diff
@@ -95,6 +96,17 @@ func (userEvaluator) Validate(scope string) error {
 	return fmt.Errorf("unrecognized scope: %v", scope)
 }
 
+func (userEvaluator) Describe(scope string) string {
+	switch scope {
+	case UserIndicator + UserInfo:
+		return "information about you, including username, identity names, and group membership"
```

Comment:
> If this information is going to be returned to a user, should it be properly capitalized and punctuated?
> 
> Also, grammar-nit; needs a colon (but I agree on the use of the Oxford Comma):
> "Information about you, including: username, identity names, and group membership."

@jwforres @fabianofranz I think that @liggitt's intent is for this to be a human readable string you can present to a user to describe a scope.  I don't know precisely how he intends for you to get the information in the console, but in the CLI this is pretty easy to get.  Should it be a complete sentence?